_ont_xgs/ont-fs-XGS-ONU-25-20NI.md

```diff
@@ -0,0 +1,589 @@
+---
+title: ONT FS.com Generic Compatible XGSPON Stick ONU with MAC SFP+ (XGS-ONU-25-20NI)
+has_children: true
+layout: default
+parent: ONT FS.com
+---
+
+# Hardware Specifications
+
+|                  |                                         |
+| ---------------- | --------------------------------------- |
+| Vendor/Brand     | FS                                      |
+| Model            | XGS-ONU-25-20NI                         |
+| ODM              | CIG                                     |
+| ODM Product Code | XG-99S                                  |
+| Chipset          | Cortina CA8271A                         |
+| Flash            | MX35LF1GE4AB 128MB                      |
+| RAM              | 128MB                                   |
+| CPU              | Taroko V0.2 (MIPS)                      |
+| CPU Clock        | 500MHz                                  |
+| Bootloader       | SATURN uboot                            |
+| System           | Custom Linux by Cortina (Saturn SDK) based on Kernel 4.4 Saturn-sfpplus-r1                                                   |
+| 2.5GBaseX        | Yes                                     |
+| XGMII/XSGMII     | Yes                                     |
+| Optics           | SC/APC                                  |
+| IP address       | 192.168.100.1                           |
+| Web Gui          |                                         |
+| SSH              | No                                      |
+| Telnet           | ✅ user `PON S/N`, password `8digit HMAC-MD5 (PON S/N uppercase)` |
```

Comment:
```suggestion
| Telnet           | ✅    |
```
telnet dosen't have password, this credential are for the enable password

Reply:
Telnet won't need a password when connecting, sure, but the user/password is immediately asked after connection, before the prompt is available.

Reply:
@simonebortolin 
Even with my Stick, login was requested when telneting. (And it wasn't required by enable)
Please let me know if the behavior is different, as it may be a different firmware.
```
telnet 192.168.100.1
Login as: aaa
Password:
Login incorrect
 - connection with the host has been lost. -
telnet 192.168.100.1
Login as: GPON2350004b
Password:UzwugGYT
ONT>enable
#ONT>
```

Reply:
@alvgalrus @YuukiJapanTech yes but that is not the telnet password, it is the enable password which is common to ssh/telnet/serial, so for consistency with CIG GPON we do not put it there